package.json

```diff
@@ -15,10 +15,12 @@
     "clean": "rimraf dist storybook-static test",
     "publish-ghpages": "mkdir -p build-storybook && npm run build-storybook && gh-pages -d storybook-static",
     "cp": "cp package.json ./dist/package.json && cp README.md ./dist/README.md ",
-    "prepare": "npm run clean",
+    "prepare": "npm run clean && husky install",
```

Comment:
Prepare is one of the reserved NPM lifecycle things, which means this command is run [every time for build + publish etc.](https://docs.npmjs.com/cli/v6/using-npm/scripts#life-cycle-scripts). Just wondering if we do want these side-effects from using `prepare`. What does it mean for Travis when it needs to build? Does it mean it'll clean & install husky again before building?

Reply:
Agree - maybe put the husky command as a "postinstall" script?